src/config/dynamic-theme-fixes.config

```diff
@@ -528,6 +528,54 @@ a[href="/product"]
 
 ================================
 
+bitwit.tech
+
+CSS
+.svg-primary {
+    fill: #375D69 !important;
+}
+.svg-primary-light {
+    fill: #4A7F8F !important;
+}
+.svg-primary-dark {
+    fill: #28444D !important;
+}
+.svg-secondary {
+    fill: #B88399 !important;
+}
+.svg-secondary-light {
+    fill: #E8A9C4 !important;
+}
+.svg-secondary-dark {
+    fill: #825F6E !important;
+}
+.svg-light {
+    fill: #ADCED9 !important;
+}
+.svg-outline {
+    fill: none !important;
+    stroke: #000000 !important;
+}
+.svg-primary-outline {
+    fill: none !important;
+    stroke: #ADCED9 !important;
```

Comment:
Have a look with `stroke: ${originial value} !important;`

Reply:
The stroke color changes depending on prefers-color-scheme. I wanted the style to be consistent no matter what, so that's why I specified the values here.